source/includes/steps-github-deploy-rollback.yaml

```
@@ -0,0 +1,54 @@
+title: View Your App's Deployment History 
+ref: view-app-deployment-history
+content: |
+  In order to rollback to a prior deploy, you'll need to navigate to your 
+  {+app+} Deployment History page.
+
+  Click :guilabel:`Deployment` in the {+leftnav+} of the {+ui+}. Select the 
+  :guilabel:`History` tab.
+
+  You'll see the 25 most recent application deployments. The 
+  :guilabel:`Change Origin` column shows whether the deploy originated from
+  changes in the UI, the linked GitHub repository, the Admin API, or the CLI.
+
+  .. figure:: /images/realm-app-deploy-history.png
+     :alt: Deployment History screen with Realm, Deployment, and History tabs circled
+---
+title: Re-Deploy a Prior Deploy
+ref: re-deploy-prior-deploy
+content: |
+  Once you have selected a prior version to rollback to, press the 
```

```suggestion
  Select a prior version to rollback to and then press the 
```

nit: technically we haven't told them to pick one yet!